general-superstaq/general_superstaq/service.py

```diff
@@ -124,21 +124,24 @@ def submit_qubo(
         target: str,
         repetitions: int = 1000,
         method: Optional[str] = None,
-        maxout: Optional[int] = 1000,
+        max_solutions: Optional[int] = 1000,
```

Comment:
`Optional[<x>]` is just an alias for `Union[<x>, None]` - this parameter should always be an integer so it's not necessary here 
```suggestion
        max_solutions: int = 1000,
```

Reply:
done